examples/chat.pl

```diff
@@ -0,0 +1,34 @@
+#
+# Minimal single-process WebSocket chat application for browser testing
+#
+use Mojolicious::Lite -signatures;
+use Mojo::EventEmitter;
+
+helper events => sub { state $events = Mojo::EventEmitter->new };
```

Comment:
This example probably doesn't work under prefork... Can an example be added that is compatible with prefork?